lib/generators/active_admin/resource/templates/admin.rb

```diff
@@ -1,5 +1,18 @@
 ActiveAdmin.register <%= class_name %> do
 
+# Available Filters
+#<% class_name.constantize.new.attributes.keys.each do |attr| %>
+# filter :<%= attr.gsub(/_id$/, '') %><% end %>
```

Comment:
@timoschilling I'll probably extract the functionality out into a common class. I didn't know about `-%>`, thanks!